Android/Fundamentals/intro-to-android/hello-android.md

```diff
@@ -0,0 +1,53 @@
+# hello-android
+author: tlawson
+
+levels:
+
+  - beginner
+
+type: normal
+
+category: how to
+
+links:
+
+  - '[link to learn more](https://enki.com)'
+
+---
+## Content
+
+We are going to create a simple Hello Android app using the following directions below:
+Open Android Studio and on the Welcome menu, click on Start a New Android Studio Project. A new project window opens and the following information will appear to include: 
+
+*	Application Name. This will appear to users. Example name it First App. 
+*	Company Domain. This is the name of your company. This is used to generate the package name below the field  
+  and a package namespace for your app. 
+*	Project Location. This is the location in which you want to place your project in. Click Next. 
+* Leave Phone and Table checked and leave the Minimum SDK at the default. Click Next. 
+* Select Empty Activity selected and click Next.
+*	Leave all the details for the activity in their default state. Click Finish.
+*	Android Studio may take at least a minute or two to set up. 
+*	Hit the run button to test your app (looks like a green play button).
+*	If there are no errors, you will be prompted to run on a specific device or on an emulator. 
+
+In the future insights, we will get more in depth of what each section means and how to use it. 
+
+---
+## Practice
+
+What is the minimum SDK used?
```

Comment:
I think the answer to this will vary depending on which Android Studio version the user installs. It also depends on specific settings that the user can change. Perhaps instead the question can be phrased to "What does the minimum SDK option mean when you're creating a new project?"